src/website/views/components/DownloadData/DownloadData.tsx

```diff
@@ -0,0 +1,28 @@
+const downloadSensorData = async (sensorType) => {
+    try {
+      const response = await fetch(`http://localhost:3005/api/${sensorType}`);
```

Comment:
Please use variables `SERVER_HOST` and `SERVER_PORT` from `.env.local`. 

Similar example shown here: https://github.com/UBCSailbot/sailbot_workspace/blob/main/src/website/stores/GPS/GPSService.ts#L9